book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def rentals(person, date)
+    Rental.new(date, self, person)
+  end
+
+  attr_accessor :title, :author
```

Comment:
- [ ] A rental should be written/read in the book class. Kindly add rentals to the attr_accessor.